contracts/libs/ModExp.sol

```diff
@@ -1,6 +1,11 @@
 pragma solidity ^0.5.15;
 
-library modexp_3064_fd54 {
+/**
+    @notice Compute $input^(N - 2) mod N$ using Addition Chain method.
+    Where     N = 0x30644e72e131a029b85045b68181585d97816a916871ca8d3c208c16d87cfd47
+    and   N - 2 = 0x30644e72e131a029b85045b68181585d97816a916871ca8d3c208c16d87cfd45
```

Comment:
Almost forgot what this number is. It's hard to deduct from the library name, so adding doc strings here.